doc/reference/std/misc/README.md

```diff
@@ -13,6 +13,7 @@ We can conceptually (and roughly) organize them in related group of modules:
   - [:std/misc/rbtree](rbtree.md)
   - [:std/misc/lru](lru.md)
 - Concurrency Primitives:
+  - [:std/misc/atom](atom.md)
```

Comment:
you nees to ads it to the vuepress config.js too so that it is searchable.

Reply:
I can't find anything like that in the repo. Path?

Reply:
doc/.vuepress

Reply:
Aha, my search was skipping the `.` directories. Thanks!